nubilanding/src/components/Header/styles.module.scss

```
@@ -0,0 +1,48 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:400,500|Rubik:300,400,500&display=swap");
+$color-fondo: #eeeeee;
+$color-letra: #214960;
+$color-boton: #567efb;
+
+header {
+  display: flex;
+  width: 100%;
+  overflow: hidden;
+  background-color: $color-fondo;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+nav ul {
+  display: flex;
+  
+}
+.Registrate{
+  color: $color-boton;
+  margin-right: 10px;
+  font-family: "Roboto";
+  font-size: 14px;
+  font-weight: 500;
+  background-color: transparent;
+  border: transparent;
+}
+.Iniciarsesion{
```

fijarte los espacios cuando termina y empieza una clase de estilos, y fijarte que estamos usando notacion camelCase. Iria de la siguiente forma en este caso: 
```suggestion
.iniciarSesion{
```
La primer palabra siempre en minuscula y las cada ves que empieza una nueva con mayuscula